Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -52,6 +63,12 @@
 	@Autowired
     private Environment env;
 
+	@Autowired
+	@Qualifier("conceptPowerService")
+	private IService service;
+
+	private static final int MAX_JSON_RESULTS = 5;

```

Comment:
I wouldn't put a max result number. There are some words that have way more than 5 results and there wouldn't be any way to retrieve those concepts if you limit it to 5 (as they have the exact same word).